repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -3357,57 +3348,84 @@ public List<String> deleteClassificationPropagation(String entityGuid, String cl
         }
     }
 
-    public void deleteClassificationOnlyPropagation(Set<String> deletedEdgeIds) throws AtlasBaseException {
+    public void deleteClassificationOnlyPropagation(String deletedEdgeId, String classificationVertexId) throws AtlasBaseException {
```

Comment:
I think we can't release this directly considering the backward compatibility
In production, there is a high possibility of having PENDING tasks with a list of GUIDs (which were create before releasing this change)

Reply:
addressed. Now the task is backward compatible